### PR TITLE
convo 4: simple verbs

### PR DESCRIPTION
**Pull request by ianwremmel**
Relates to #63

### CODE REVIEW

**Comment by mwegman** [comment]
LGTM

File: packages/plugin-conversation/src/conversation.js [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+    }
+
+    if (target) {
+      activity.target = target;
+    }
+
+    return activity;
+  },
+
+  get(conversation, options) {
```
Comment (Josh-Dykstra) [inline packages/plugin-conversation/src/conversation.js]: If there's any function that could probably use some docs it's this one :) maybe you did it in another PR though.

File: packages/plugin-conversation/src/conversation.js [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+  },
+
+  prepareConversation(conversation) {
+    const c = defaults(pick(conversation, `id`, `url`, `objectType`, `kmsResourceObjectUrl`, `defaultActivityEncryptionKeyUrl`), {
+      objectType: `conversation`
+    });
+    return c;
+  },
+
+  prepare(activity, params) {
```
Comment (Josh-Dykstra) [inline packages/plugin-conversation/src/conversation.js]: Just a thought about the function names in this module. I know this is how we've had things for awhile, but I wonder if we should change functions like `prepare()`, `submit()` or `expand()` to be `prepareActivity()` etc. and have `prepareConversation()` be `prepare()`. Seems like a first time user might expect `spark.conversation.prepare()` to prepare a conversation, and at the moment it's for preparing an activity. 

In reality I don't really expect most of these functions to be called outside this module, so maybe it's fine. Just wondering what you thought.

File: packages/plugin-conversation/src/interceptor.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+   * @see Interceptor#onRequest
+   * @param {Object} options
+   * @returns {Promise<Object>}
+   */
+  onRequest(options) {
+    if (!options.method || options.method.toUpperCase() === `GET`) {
+      return options;
+    }
+
+    return this.spark.device.isSpecificService(`conversation`, options.service || options.uri)
```
Comment (Josh-Dykstra) [inline packages/plugin-conversation/src/interceptor.js]: Nice. So for each new plugin that needs to be encrypted we'll just make a new interceptor that's similar to this one.

File: packages/plugin-conversation/src/conversation.js (hunk 1 above)
Comment (ianwremmel) [inline packages/plugin-conversation/src/conversation.js]: Yea, the final PR (not done yet) is going to cover docs.

File: packages/plugin-conversation/src/conversation.js (hunk 2 above)
Comment (ianwremmel) [inline packages/plugin-conversation/src/conversation.js]: good point; i've made a note to handle it in one of the later PRs (it'll really mess up the rebase for the already pending PRs)

**Pull request merged**